common/tasks/mount_overlay.yml

```diff
@@ -12,10 +12,20 @@
     - "{{ overlayfs_lower_dir }}"
     - "{{ overlayfs_upper_dir }}"
     - "{{ overlayfs_build_root }}"
+    - "{{ overlayfs_work_dir }}"
 
-- name: Mount overlay
+- name: Mount overlayfs
+  when: kernel_major == "3" and kernel_minor == "16" 
```

Comment:
The target kernel version for the discovery image is 3.13,  these checks are for the travis hosts running kernel version.